docs/development/process.md

```diff
@@ -27,7 +27,10 @@ Hotfixes are usually only maintained for the latest minor release as well as the
 
 The validation process for new releases usually takes a couple of days and includes the following steps:
 
-1. `master` (or latest `release-*` branch) is deployed to a development landscape that already hosts some existing seed and shoot clusters.
+1. If a new
+   1. minor release `vX.Y.0` shall be cut then a new branch `release-vX.Y` is created based on `master`. [*]
```

Comment:
Should we describe that the VERSION file in the master needs to be manually bumped?

Reply:
Yeah, we can do. OTOH, shall we "revert" this process again (at least for the time being and given the many issues with the release notes)? I like it, but our CI/CD infrastructure seems to not be ready yet.

Reply:
If we do not find a quick and easy workaround, yes.